lib/node_modules/@stdlib/namespace/alias2pkg/data/data.csv

```diff
@@ -1043,6 +1192,7 @@ base.strided.dvarianceyc,"@stdlib/stats/base/dvarianceyc"
 base.strided.dvarm,"@stdlib/stats/base/dvarm"
 base.strided.dvarmpn,"@stdlib/stats/base/dvarmpn"
 base.strided.dvarmtk,"@stdlib/stats/base/dvarmtk"
+base.strided.functionObject,"@stdlib/strided/base/function-object"
```

Comment:
@Planeshifter This shouldn't be exported. It is an include path. Why did you export this?